### ISSUE TITLE
support for header in Chinese in new fonts(v1.5.0)

### ISSUE THREAD

**Issue by AiurArtanis** [damiankorcz/Prism-Theme#34]
@damiankorcz hello again

# Advice

the default font in **Prism-Theme** is changed in v1.5.0, and it seems not suitable for header in Chinese characters as before. here's the comparison between v1.5.0 and preious:
![image](https://user-images.githubusercontent.com/98091664/152716713-6cd4936f-8e6a-47d9-b907-1bff4bb6b919.png)
_v1.5.0_
![image](https://user-images.githubusercontent.com/98091664/152716605-e8dceb09-972b-4b96-9267-1a9681ff7902.png)
_v1.3.0_
header Chinese characters in new fonts loses the **Bold** feature. 
(i try to replace the Font Imports in css file with the preious version, and it doesn' t work😂)

btw, the color of editor background also changes and i think the preious is better(light gray)

## Environment

os: win 10
ob ver: 0.13.19
prism ver: v1.5.0

## Last

v1.5.0 brings lots of new feature, it is cool but it is difficult to me to get used to

**Comment by AiurArtanis** [comment]
maybe it's a good idea to add the customization of the editor bg color, and i change the setting as before
![image](https://user-images.githubusercontent.com/98091664/152719329-b3cc37e1-ff61-43b8-b7c4-9e8c1c61976b.png)

**Comment by damiankorcz** [comment]
This is a strange issue since the new fonts and the newest insider version of Obsidian is using a newer version of Chrome which introduces a regression where if the default font weight of a header is changed in the CSS it causes the emoji to appear the same colour as the text like this:
![image](https://user-images.githubusercontent.com/7290097/152719679-97b84ea0-e585-4af1-aba7-38c7966cc891.png)

Instead of appearing correctly coloured like this:
![image](https://user-images.githubusercontent.com/7290097/152719736-017fcf7e-5abc-4eec-818a-2837f349b0f1.png)

Since the theme uses Variable fonts now, I've replaced the `font-weight: bold;` with `font-variation-settings: "wght" 800;` which resolved this issue for Latin characters at least but as you pointed out seems to not apply to Chinese characters.
![image](https://user-images.githubusercontent.com/7290097/152719897-4f4ee0af-0441-42fb-864f-a123d3212360.png)

I look into this and see if I can get it to work correctly.

As for the background colour of the notes, I got the opposite feedback from @claremacrae over on Discord and I personally didn't mind changing it back to a white background. This change also required some colour changes to other elements like code blocks, quotes, etc. to contrast against the background. 

I'll add a feature to toggle between the two in the next update.

**Comment by AiurArtanis** [comment]
it's weird that `font-variation-settings` is suppose to work correctly(but not) on Chinese character after i consult relevant tutorials

**Comment by AiurArtanis** [comment]
this snippet does work and it's quite a clever method as for a tmp fix. 
then i wait for ob fix the font rendering bug, anyway, thanks😉

**Comment by AiurArtanis** [comment]
> once Obsidian reaches the Chrome version which fixes the emoji issue.

i just recieve the newest update of Obsidan, and it seems that this issue is solved
![image](https://user-images.githubusercontent.com/98091664/158326173-35555170-fb03-49cf-863c-eb78800cfa6d.png)

**Comment by damiankorcz** [comment]
Hi @AiurArtanis,
I'm not sure if this issue has been resolved now with v2.0.0. Could you check for me and let me know if further fixes are needed or if this issue can be closed. Thank you!

**Comment by AiurArtanis** [comment]
> Could you check for me and let me know if further fixes are needed or if this issue can be closed

i tried to close the snippet you mentioned
```css
div.cm-s-obsidian .cm-header,
div.markdown-preview-view :is(h1, h2, h3, h4, h5, h6) {
	font-weight: 800;
	font-variation-settings: unset;
}
```
and it works well. thank u for v2.0.0 which is a huge update, and i start to check other new feature😁
